src/lib/recoil/jobs.js

```diff
@@ -0,0 +1,90 @@
+import {atom, noWait, selector, selectorFamily} from "recoil";
+import {httpState, optionsState} from "./base";
+import {careerState, careersParamsState} from "./career";
+
+export const careerJobsState = atom({key: "career-modal-jobs", default: {fetching: false, jobs: []}});
+export const jobOptionsState = atom({key: "job-options", default: {}});
+
+const jobsPathState = selector({
+  get: ({get}) => {
+    const career = get(careerState);
+    const options = get(optionsState).career || {};
+
+    if(options.jobsPath) { return `${options.jobsPath}/${career.id}/jobs`; }
```

Comment:
I think maybe `options.jobsPath` should be a function that takes the careerID and returns the path

Reply:
Good Idea! Ok updated!